apps/linkedin-to-notion/src/contents/linkedin-data-cleaners.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Basic function to split first and last name in common case ie. they are split by a space
+ * @param fullName Full name eg. 'Gauvain Thery'
+ * @returns an object with a split between first and last names
+ */
+export const splitName = (fullName: string): { firstName?: string; lastName?: string } => {
+  const names = fullName.split(' ');
+  if (names.length === 1) {
+    return {
+      firstName: names[0],
+    };
+  }
+  return {
+    firstName: names[0].trim(),
+    lastName: names[names.length - 1].trim(),
```

Comment:
Here, you've taken the last element of `names` as the last name, but what would happen with *Jean De La Rochebrochard* for instance? That's why I think this is better:

```suggestion
    lastName: names.slice(1).join(" ").trim(),
```